application/controllers/admin/authentication.php

```diff
@@ -288,7 +288,7 @@ private function _doRedirect()
     * Renders template(s) wrapped in header and footer
     *
     * @param string $sAction Current action, the folder to fetch views from
-    * @param string|array $aViewUrls View url(s)
+    * @param string $aViewUrls View url(s)
```

Comment:
Not true : see http://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.param.pkg.html multiple datatype use string|array .